test/WebJobs.Script.Tests/Security/SecretManagerTests.cs

```diff
@@ -1105,7 +1188,7 @@ private Mock<IKeyValueConverterFactory> GetConverterFactoryMock(bool simulateWri
             return mockValueConverterFactory;
         }
 
-        private SecretManager CreateSecretManager(string secretsPath, ILogger logger = null, IMetricsLogger metricsLogger = null, IKeyValueConverterFactory keyConverterFactory = null, bool createHostSecretsIfMissing = false, bool simulateWriteConversion = true, bool setStaleValue = true)
+        internal SecretManager CreateSecretManager(string secretsPath, ILogger logger = null, IMetricsLogger metricsLogger = null, IKeyValueConverterFactory keyConverterFactory = null, bool createHostSecretsIfMissing = false, bool simulateWriteConversion = true, bool setStaleValue = true)
```

Comment:
Is this change necessary? Didn't see it being accessed from other classes?

Reply:
Whoops, neglected to reset after deciding to add tests in-place.